scripts/default.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+let datasource = 'test';
+let region = 'eu-central-1';
+let components = [
```

Comment:
Hmm. Finnes det noen mulighet for å gjøre denne listen litt mer dynamisk? "Garanterer" at vi glemmer å oppdatere den når vi legger til en ny komponent...